### PR TITLE
Unstable/main

### PR DESCRIPTION
**Pull request by Sanduhr32**
Fixes # .

- [x] I have tested my changes.
- [ ] I didn't test my changes. I will do that in the following 7 days and agree that this Pull request will be closed if I don't test it in the named time span.
- [x] I know the language good enough  for providing high quality code.

# Changes proposed in this pull request:
- Exception handling
- WebUtils changes (yes again)

---
## Notes:
- [x] Will update build gradle build version and set unstable to false

---
### Author
  __*Sanduhr32*__

### CODE REVIEW

File: src/main/java/ml/duncte123/skybot/BotListener.java [hunk 1]
```diff
@@ -209,7 +210,20 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
     @Override
     public void onReady(ReadyEvent event){
         logger.info("Logged in as " + String.format("%#s (Shard #%s)", event.getJDA().getSelfUser(), event.getJDA().getShardInfo().getShardId()));
-        
+
+        if (Settings.isUnstable) {
+            if (event.getJDA().getSelfUser().getIdLong() != 210363111729790977L) {
```
Comment (duncte123) [inline src/main/java/ml/duncte123/skybot/BotListener.java]: These 2 if-statements can be merged?

File: src/main/java/ml/duncte123/skybot/BotListener.java [hunk 2]
```diff
@@ -306,6 +320,17 @@ public void onGuildJoin(GuildJoinEvent event) {
             return;
         }
         Guild g = event.getGuild();
+        if (Settings.isUnstable) {
+            if (event.getJDA().getSelfUser().getIdLong() != 210363111729790977L) {
```
Comment (duncte123) [inline src/main/java/ml/duncte123/skybot/BotListener.java]: These 2 if-statements can be merged?

Wait, duplicated code? make a method.

File: src/main/java/ml/duncte123/skybot/utils/MessageUtils.java [hunk 3]
```diff
@@ -58,6 +59,18 @@ public static void sendError(Message message) {
         message.addReaction("❌").queue(null, CUSTOM_QUEUE_ERROR);
     }
 
+    @DocumentationNeeded
```
Comment (duncte123) [inline src/main/java/ml/duncte123/skybot/utils/MessageUtils.java]: I would like documentation for all those methods

Comment (Sanduhr32) [reply src/main/java/ml/duncte123/skybot/utils/MessageUtils.java]: No you dont get it rn. Thats why the annotation exists

File: src/main/java/ml/duncte123/skybot/BotListener.java (hunk 2 above)
Comment (Sanduhr32) [reply src/main/java/ml/duncte123/skybot/BotListener.java]: Sure......

File: src/main/java/ml/duncte123/skybot/utils/MessageUtils.java (hunk 3 above)
Comment (duncte123) [reply src/main/java/ml/duncte123/skybot/utils/MessageUtils.java]: it's better to do it now, don't delay the boring tasks

Comment (Sanduhr32) [reply src/main/java/ml/duncte123/skybot/utils/MessageUtils.java]: If you want it good docs now do it. I will just add scrap.

**Pull request merged**